CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 ## [Unreleased]
 
 ## [4.2.0] - 2020-05-06
+* Modified const name in wp-config-project.php from ES_ENV to EB_ENV to be consistent with the rest of the project.
```

Comment:
you can' add stuff to the release that is all-ready out. Please add this to the Unreleased heading.